dill/_objects.py

```diff
@@ -111,6 +109,7 @@ class _Struct(ctypes.Structure):
         pass
     _Struct._fields_ = [("_field", ctypes.c_int),("next", ctypes.POINTER(_Struct))]
 _filedescrip, _tempfile = tempfile.mkstemp('r') # deleted in cleanup
+os.close(_filedescrip)
```

Comment:
When running test_objects.py both cases gave the same results
1. close file descriptor & delete tempfile
2. do neither

result:

```bat
fails: CLibraryLoaderType <class 'ctypes.LibraryLoader'>
fails: TextWrapperType <type 'file'>
fails: BufferedRandomType <type 'file'>
fails: BufferedReaderType <type 'file'>
fails: PyTextWrapperType <class '_pyio.TextIOWrapper'>
fails: PyBufferedRandomType <class '_pyio.BufferedRandom'>
fails: PyBufferedReaderType <class '_pyio.BufferedReader'>
fails: TarFileType <class 'tarfile.TarFile'>
fails: FileType <type 'file'>
fails: ListIteratorType <type 'listiterator'>
fails: TupleIteratorType <type 'tupleiterator'>
fails: XRangeIteratorType <type 'rangeiterator'>
fails: CycleType <type 'itertools.cycle'>
fails: TemporaryFileType <type 'instance'>
fails: PermutationsType <type 'itertools.permutations'>
fails: CombinationsType <type 'itertools.combinations'>
fails: RepeatType <type 'itertools.repeat'>
fails: CompressType <type 'itertools.compress'>
```